src/screens/SUE/SueReportScreen.tsx

```diff
@@ -42,16 +42,24 @@ type TRequiredFields = {
   };
 };
 
+const keyMapping = {
+  [INPUT_KEYS.SUE.NAME]: INPUT_KEYS.SUE.FIRST_NAME,
+  [INPUT_KEYS.SUE.FAMILY_NAME]: INPUT_KEYS.SUE.LAST_NAME
+};
+
+export const getMappedKey = (inputKey: string) => keyMapping[inputKey] || inputKey;
+
 const sueProgressWithRequiredInputs = (
-  fields: TRequiredFields,
+  requiredFields: TRequiredFields,
   geoMap: { locationIsRequired: boolean; locationStreetIsRequired: boolean },
   progress: TProgress[]
 ): TProgress[] => {
   const requiredInputs: { [key: string]: boolean } = {};
 
-  if (fields?.contact) {
-    for (const field in fields.contact) {
-      requiredInputs[field] = fields.contact[field];
+  if (requiredFields?.contact) {
+    for (const field in requiredFields.contact) {
+      const mappedKey = getMappedKey(field);
+      requiredInputs[mappedKey] = requiredFields.contact[field];
```

Comment:
i guess `requiredInputs[getMappedKey(field)] = requiredFields.contact[field];` should be fine